trade/trader_stakecube.js

```diff
@@ -180,16 +184,16 @@ module.exports = (
         return undefined;
       }
 
-      let orders;
+      let scData;
 
       try {
-        orders = await stakeCubeApiClient.getOrders(pair_.pair);
+        scData = await stakeCubeApiClient.getOrders(pair_.pair);
 
-        if (orders.errorMessage) {
-          if (orders.errorMessage === 'no data') {
+        if (scData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of empty result, it will be `[]`.